javaparser-core/src/main/java/com/github/javaparser/Position.java

```diff
@@ -3,8 +3,8 @@
 import com.github.javaparser.ast.Node;
 
 public class Position {
-    private int line;
-    private int column;
+    public final int line;
```

Comment:
making the field public instead of using a getter we cannot later change the way it is stored
I guess it could be ok in this case